src/redist/targets/GenerateLayout.targets

```diff
@@ -109,15 +105,8 @@
       </BundledLayoutPackage>
 
       <BundledLayoutPackage Include="MicrosoftNetCoreAppHostPackNupkg">
-        <PackageName>runtime.$(SharedFrameworkRid).Microsoft.NETCore.DotNetAppHost</PackageName>
-        <PackageVersion>$(NetCoreAppTargetingPackVersion)</PackageVersion>
-        <TargetFramework>$(TargetFramework)</TargetFramework>
-        <RelativeLayoutPath>packs/%(PackageName)/%(PackageVersion)</RelativeLayoutPath>
-      </BundledLayoutPackage>
-
-      <BundledLayoutPackage Include="MicrosoftNetCoreAppHostPackNupkg_Altnernate" Condition="'$(AlternateAppHostRid)' != ''">
-        <PackageName>runtime.$(AlternateAppHostRid).Microsoft.NETCore.DotNetAppHost</PackageName>
-        <PackageVersion>$(NetCoreAppTargetingPackVersion)</PackageVersion>
+        <PackageName>Microsoft.NETCore.DotNetAppHost</PackageName>
```

Comment:
Is an identity apphost package still used for something? I was assuming only the `Microsoft.NETCore.App.Host.<rid>` Packs would be used and the old package would be deletable as far as core-sdk is concerned.

Reply:
Davis is right, this should be `Microsoft.NETCore.App.Host.<rid>` now

Reply:
Done.